openfga_sdk/sync/__init__.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+
+# flake8: noqa
+"""
+   Python SDK for OpenFGA
+
+   API version: 0.1
+   Website: https://openfga.dev
+   Documentation: https://openfga.dev/docs
+   Support: https://discord.gg/8naAwJfWN6
+   License: [Apache-2.0](https://github.com/openfga/python-sdk/blob/main/LICENSE)
+
+   NOTE: This file was auto generated by OpenAPI Generator (https://openapi-generator.tech). DO NOT EDIT.
+"""
+
+from openfga_sdk.sync.client.client import OpenFgaClient
+from openfga_sdk.sync.api_client import ApiClient
+
+from openfga_sdk.client.configuration import ClientConfiguration
+import openfga_sdk.configuration as configuration
+from openfga_sdk.configuration import Configuration
+
+from openfga_sdk.exceptions import OpenApiException
+from openfga_sdk.exceptions import FgaValidationException
+from openfga_sdk.exceptions import ApiValueError
+from openfga_sdk.exceptions import ApiAttributeError
+from openfga_sdk.exceptions import ApiKeyError
+from openfga_sdk.exceptions import ApiException
+from openfga_sdk.exceptions import NotFoundException
+from openfga_sdk.exceptions import UnauthorizedException
+from openfga_sdk.exceptions import ForbiddenException
+from openfga_sdk.exceptions import ServiceException
+from openfga_sdk.exceptions import ValidationException
+from openfga_sdk.exceptions import AuthenticationError
+from openfga_sdk.exceptions import RateLimitExceededError
```

Comment:
@booniepepper can we remove anything here that's not part of the sync module? Leaving just `ApiClient` and `OpenFgaClient`

Reply:
Can do, this was necessary to make tests "just work" but I can update the tests too

Reply:
Removed what I can in https://github.com/openfga/python-sdk/pull/40/commits/2f313f464e7007d0cd4b4489a33051bba1de1882